forwards_paper.tex

```diff
@@ -64,25 +99,38 @@
 
 
 \begin{abstract}
-    In this paper we describe how to
-    efficiently record the entire genetic history of a population
-    during a forwards-time, individual-based population genetics simulation.
-    This dramatically reduces the computational burden of tracking individual genomes
-    by simulating only those loci that may affect reproduction (those having non-neutral variants),
-    and recording history as a \emph{succinct tree sequence} as introduced in the software package \msprime,
+    In this paper we describe how to efficiently record the entire genetic history of a population
+    in forwards-time, individual-based population genetics simulations with arbitrary breeding models,
+    population structure and demography.
+    This approach dramatically reduces the computational burden of tracking individual genomes
+    by allowing us to simulate only those loci that may affect reproduction
+    (those having non-neutral variants).
+    The genetic history of the population is recorded as a succinct tree sequence
+    as introduced in the software package \msprime,
     on which neutral mutations can be quickly placed afterwards.
-    We implement the method in two popular forwards-time simulation frameworks,
-    which speeds up large, whole-genome simulations by one to two orders of magnitude.
-    In addition to speed, the method has several advantages:
+    Recording the results of each breeding event
+    results in a linear growth in the amount of information that must be stored.
+    However, there is a great deal of redundancy in this information,
```

Comment:
s/results in a linear growth in the amount of information that must be stored./requires storage that grows linearly./